Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -121,41 +156,24 @@ public struct Configuration: Equatable {
         do {
             let yamlContents = try NSString(contentsOfFile: fullPath,
                 encoding: NSUTF8StringEncoding) as String
-            let dict = try YamlParser.parse(yamlContents)
-            if !silent {
-                queuedPrintError("Loading configuration from '\(path)'")
+            do {
+                let dict = try YamlParser.parse(yamlContents)
+                if !quiet {
+                    queuedPrintError("Loading configuration from '\(path)'")
+                }
+                self.init(dict: dict)!
+                configPath = fullPath
+                return
+            } catch {
+                queuedPrintError("Error parsing YAML: \(error)")
```

Comment:
It would be better if YAML parsing error would contains location information.
But it seems YamlSwift does not have such functions.